### PR TITLE
Fall back to default popup value for extension if tab specific value not available

### PR DESCRIPTION
**Pull request by bsclifton**
Fixes https://github.com/brave/browser-laptop/issues/10224

To be merged into Muon 4.3.x (for use with browser-laptop 0.18.x hotfix)

Auditors: @jonathansampson

### CODE REVIEW

File: lib/browser/api/browser-actions.js [hunk 1]
```diff
@@ -62,6 +64,8 @@ module.exports = {
   },
 
   getPopup: (extensionId, details) => {
-    return getBrowserAction(extensionId, details).popup
+    const action1 = getBrowserAction(extensionId, details)
+    const action2 = getBrowserAction(extensionId, {})
```
Comment (jonathansampson) [inline lib/browser/api/browser-actions.js]: We might consider moving this call to `getBrowserAction` behind the `||` on the next line. That way, if `action1` is all that was needed, we don't make an unnecessary second call.

Comment (bsclifton) [inline lib/browser/api/browser-actions.js]: @jonathansampson updated per your feedback 😄  Makes the change look a little smaller too

**Comment by bsclifton** [comment]
@jonathansampson @kevinlawler ready for re-review after feedback implemented 😄 

Also, I assigned milestone + labels

File: lib/browser/api/browser-actions.js [hunk 2]
```diff
@@ -62,6 +64,6 @@ module.exports = {
   },
 
   getPopup: (extensionId, details) => {
-    return getBrowserAction(extensionId, details).popup
+    return getBrowserAction(extensionId, details).popup || getBrowserAction(extensionId, {}).popup
```
Comment (bridiver) [inline lib/browser/api/browser-actions.js]: I don't think this extra call is necessary. I think all you need here is a default value for `details` in `getBrowserAction`

Comment (bridiver) [inline lib/browser/api/browser-actions.js]: I don't understand what this is supposed to be doing

File: lib/browser/api/browser-actions.js [hunk 3]
```diff
@@ -13,10 +13,12 @@ const updateBrowserAction = (extensionId, details) => {
 
 const getBrowserAction = (extensionId, details) => {
```
Comment (bridiver) [inline lib/browser/api/browser-actions.js]: just a nit that it would be better to provide a default value for details instead of passing in `{}`

**Pull request merged**